templates/zerver/help/manage-system-notifications.md

```diff
@@ -0,0 +1,49 @@
+# Manage System Notifications
+
+Zulip is able to send you notifications when you are away from Zulip in
+various ways. It can send desktop notifications when Zulip is merely out of
+focus, email digests when you are offline, and mobile notifications according
+to your preferences. In order to manage how Zulip contacts you, go to
+[notification settings](/#settings/notifications) and select the notification
+options that you want.
+
+## Stream Settings
+
+By default, Zulip does not send notifications for your subscribed streams. If
+you want to change that default and get notifications for your subscribed
+streams, go to the [system notification settings](/#settings/notifications)
+and select one or several of the options, including **Visual Desktop Notifications**,
+**Audible Desktop Notifications**, and/or **Mobile Notifications**. You
+can then choose to apply the notification settings on each of your
+subscribed streams, or only to future subscriptions. If you want to adjust 
+the notification settings on individual streams to be different from the 
+system setting that you set in system
+notifications, you can adjust the [settings on individual streams](/#streams)
+after you set the default.
+
+!!! warn ""
+     **Note:** If you change the default after adjusting
+	the notification settings on individual streams, the system setting will override the
+	previous individual setting.
+
+## Private Messages and @-Mentions
+
+By default, Zulip will send visual and audible desktop notifications for
+private messages and @-mentions, as well as email and mobile notifications for
+private messages and @-mentions when you are offline. You have the option of
+including or excluding the content of private messages from your desktop
+notifications. You also have the option of activating mobile notifications
+even when you are online.
+
+## Other notification settings
+
+--Can we rename "Other notification settings" to "Digest Email Settings"
+
+--What actually goes into email digests?
+
+--Can we move "Include content of private messages" to the emails
+--notifications in the "private messages" section
```

Comment:
This was a bug that's now been fixed in master. Thanks :).

Reply:
What I see in master is just an update in the wording on one of the settings. 

I think that it might make more sense to move that setting " Include message content in missed message emails" to be indented under "Email notifications when offline" so that it parallels "Visual desktop notifications" and it's "Include content of private messages"

![screenshot from 2018-04-10 16-52-52](https://user-images.githubusercontent.com/549661/38583090-e0b849ee-3cdf-11e8-97e4-0841bdbe5062.png)

Reply:
I could be being too picky....

Reply:
That last move doesn't make sense, because that setting applies to all emails, not just the private message ones (we're adding a "Stream emails" setting soon)